cpg_workflows/stages/large_cohort.py

```diff
@@ -52,18 +55,21 @@ def expected_outputs(self, cohort: Cohort) -> Path:
         return get_workflow().prefix / 'sample_qc.ht'
 
     def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
-        from cpg_workflows.large_cohort.dataproc_utils import dataproc_job
-        from cpg_workflows.large_cohort.sample_qc import run
+        from cpg_workflows.large_cohort import sample_qc
 
-        j = dataproc_job(
-            job_name=self.__class__.__name__,
-            function=run,
-            function_path_args=dict(
-                vds_path=inputs.as_path(cohort, Combiner),
-                out_sample_qc_ht_path=self.expected_outputs(cohort),
-                tmp_prefix=self.tmp_prefix,
-            ),
-            depends_on=inputs.get_jobs(cohort),
+        j = get_batch.new_job(
+            'Sample QC', (self.get_job_attrs() or {}) | {'tool': 'hail query'}
+        )
+        j.image(image_path('cpg_workflows'))
+        j.command(
+            query_command(
+                sample_qc,
+                sample_qc.run.__name__,
+                str(inputs.as_path(cohort, Combiner)),
```

Comment:
```suggestion
                inputs.as_str(cohort, Combiner),
```

Should work as an alternative here